lib/app.js

```diff
@@ -87,14 +87,18 @@ const MAPPINGS = {
     path: ['ds:10', 0, 1],
     fun: buildHistogram
   },
+  preregister: {
```

Comment:
I'm not sure this is true. When you just upload a new app it doesn't have reviews, and thus won't have an histogram; that doesn't mean it's a preregister

Reply:
I bet there's a boolean somewhere in there, although I admit it's going to be hard to track down. Is there somewhere we can find a list of 4 or 5 preregister apps so we can later test?

Reply:
That doesn't seem to be the case, you can check these apps (with no downloads at the moment of writing, you can find others in the "top new paid" section):

https://play.google.com/store/apps/details?id=com.ThinkFast.CublastHD
https://play.google.com/store/apps/details?id=com.JacobDunbarGames.MayorMayorPremium

Those are flagged as preregister in your branch, although they are not.

Reply:
Yes, it's true. I'm removing the `preregister` field for now, but I left the bugs fix there. I'll keep trying to track this.